src/Build/Resources/Strings.resx

```diff
@@ -527,6 +527,10 @@
     <value>MSB4019: The imported project "{0}" was not found. Confirm that the expression in the Import declaration "{1}" is correct, and that the file exists on disk.</value>
     <comment>{StrBegin="MSB4019: "}LOCALIZATION: &lt;Import&gt; should not be localized.</comment>
   </data>
+  <data name="ImportedProjectFromVSDistribution" xml:space="preserve">
+    <value>MSB4278: The imported project {0} does not exist and appears to be part of Visual Studio. This project may require MSBuild.exe and fail to build in the dotnet CLI. </value>
```

Comment:
Would still like @barofel input on this wording.

Reply:
I'm struggling a bit with the term 'project' here. I know it's technically correct (every `Import` imports an MSBuild project file, we just assign specific semantics to ones like D.B.props, D.B.targets, .csproj, etc) but I don't think users think of these files in terms of being 'projects'. So I think we need a better term here (and TBH in other places where we reference 'project', like in 'ImportedProjectNotFound` directly above.

```suggestion
    <value>MSB4278: The imported file {0} does not exist and appears to be part of a Visual Studio component. This file may require MSBuild.exe in order to be imported successfully, and so may fail to build in the dotnet CLI. </value>
```